pygmt/src/histogram.py

```diff
@@ -10,11 +10,20 @@
     A="horizontal",
     B="frame",
     C="cmap",
+    D="annotate",
+    E="bar_width",
```

Comment:
Remove underscore here, see also https://github.com/GenericMappingTools/pygmt/pull/1190#issuecomment-814451969 and https://github.com/GenericMappingTools/pygmt/pull/1119#issuecomment-815901878 (which we should document soon).

```suggestion
    E="barwidth",
```